### PR TITLE
Migrate from jupyter_sphinx_theme to sphinx_rtd_theme

### PR DESCRIPTION
**Pull request by gnestor**
Closes https://github.com/ipython/ipywidgets/issues/799

### CODE REVIEW

**Comment by gnestor** [comment]
@willingc Do you have any specific ideas about how we can improve the style of this RTD theme?

**Comment by willingc** [comment]
@gnestor I'll leave a more detailed response in the morning. Thanks! Looking forward to moving this forward.

**Comment by willingc** [comment]
Do you want me to merge this and we can iterate?

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 jupyter_client
 sphinx==1.3.6
```
Comment (willingc) [inline docs/requirements.txt]: we should update this to have the same version 1.4.5 or higher

Comment (willingc) [inline docs/requirements.txt]: also add latest version of nbsphinx (and recommonmark for md)

**Comment by SylvainCorlay** [comment]
I will test this real quick in the afternoon. I am fine with merging and iterating.

**Comment by SylvainCorlay** [comment]
Regarding the rendering of live widgets in sphinx, instead of having it done by our custom theme, we discussed with @ellisonbg and we are going to create a sphinx extension instead.

**Comment by willingc** [comment]
@SylvainCorlay Good plan to create a sphinx extension for live rendering :sunny:

**Comment by gnestor** [comment]
@SylvainCorlay I like this idea. Will this then depend on https://github.com/ipython/ipywidgets/issues/800? Do we currently have a way of converting notebooks into static HTML with live widgets?

**Comment by gnestor** [comment]
@willingc After Sylvain has tested, let's merge and iterate because this is an improvement over the currently published docs. You should be able to use this commit as reference to update the jupyter/jupyter docs.

**Comment by willingc** [comment]
@gnestor Sounds good. I would like to have a more visually appealing landing page for the jupyter/jupyter docs. After merging this, I can create a WIP PR for jupyter/jupyter and we can iterate the landing page there.

**Comment by gnestor** [comment]
@willingc Oh nevermind, it looks like the Jupyter docs have already been switched back to the RTD theme 👍

**Comment by gnestor** [comment]
Correction: the [notebook docs](https://jupyter-notebook.readthedocs.io/en/latest/) have been switched back but the [jupyter docs](https://jupyter.readthedocs.io/en/latest/index.html) are still using the jupyter theme.

**Comment by willingc** [comment]
@gnestor All of the docs for individual projects typically use the RTD theme. Jupyter/jupyter is a consolidation of the various docs to better reflect Fernando/Brian's vision earlier this year for the docs landing page.

**Comment by SylvainCorlay** [comment]
> @SylvainCorlay I like this idea. Will this then depend on #800? Do we currently have a way of converting notebooks into static HTML with live widgets?

Yes, I imagine that it will probably use that. We should have a discussion on that.

**Comment by SylvainCorlay** [comment]
@gnestor @willingc I am ok to merge but this needs a rebase.

Since the rebase was trivial, I will rebase it locally and merge so that we can iterate.

**Comment by willingc** [comment]
Thanks @SylvainCorlay and @gnestor

**Pull request closed without merging**